module-1/lab-errhand_listcomp/your-code/main.py

```diff
@@ -107,6 +112,10 @@
 #9. Find all the consonants in the sentence 'The quick brown fox jumped over the lazy dog'.
 # Use consonants as the name of the list.
 # Remember to use list comprehensions and to print your results.
+
+'''
+Seleccionaos los caracteres que son alfanuméricos y quitamos las vocales
+'''
 vowels = set("aeiouAEIOU")
```

Comment:
You don't need to run a join, just use a list with the vowels where each vowel is a string. It's simpler and makes the code more readable.